airflow/api_fastapi/core_api/datamodels/connections.py

```diff
@@ -90,8 +91,74 @@ class ConnectionBody(BaseModel):
     extra: str | None = Field(default=None)
 
 
-class ConnectionBulkBody(BaseModel):
-    """Connections Serializer for requests body."""
+class ConnectionBulkCreateAction(BaseModel):
+    """Bulk Create Variable serializer for request bodies."""
+
+    action: Literal["create"] = "create"
+    connections: list[ConnectionBody] = Field(..., description="A list of connections to be created.")
+    action_if_exists: Literal["skip", "overwrite", "fail"] = "fail"
+
+
+class ConnectionBulkUpdateAction(BaseModel):
```

Comment:
In a follow up PR, we can unify the datamodels. Basically, all the `XXXXXXBulkUpdateAction` for connections, variables etc... will have `action` and `action_if_not_exists` that are the same, we could mutualize some fields and code in common base datamodels. (for later).